spec/rails-settings-cached/setting_spec.rb

```diff
@@ -96,6 +105,11 @@
       expect(Setting.get_all).to include(:default1, :default2)
     end
 
+    it "should include namespace defaults" do
+      expect(RailsSettings::Default).to receive(:instance).and_return({ "test.default1" => 1, "test.default2" => '2', demo: 3 })
+      expect(Setting.get_all('test.')).to include(:'test.default1', :'test.default2')
```

Comment:
This is more workaround than feature.
Currently this gem has wrong logic. For me it was broken (before this patch). 
In your version, we can select `{test: { fff: {} } }` data by 'test.fff', but we cannot store it same way (it will be stored as different path 'test.fff').
So, we must get it to work in useful way (as minimum) or it must have multi-direction storage/selection features ( as I offered).